src/dagmc/test/CMakeLists.txt

```diff
@@ -26,6 +26,15 @@ setup_test(dagmc_pointinvol_test cpp ${DRIVER} "${LINK_LIBS}")
 setup_test(dagmc_rayfire_test    cpp ${DRIVER} "${LINK_LIBS}")
 setup_test(dagmc_simple_test     cpp ${DRIVER} "${LINK_LIBS}")
 
+if(OPENMP_BUILD)
```

Comment:
Is `OPENMP_BUILD` a DAGMC CMake variable? Should we add it as an option in the main CMakeLists file?

Reply:
Since the threadmanager does not itself depend on OpenMP this was only so that the test actually uses it. So I'll defer on that one, all the -D options are all global anyhow, so if you set 
```
cmake -DOPENMP_BUILD=ON ..... 
```
Then its throughout all the Cmakefiles anyhow